### PR TITLE
Doc: Fix invalid link formatting

### PR DESCRIPTION
**Pull request by karenzone**
We need these doc fixes so that we can stop propagating these errors in generated files (3 source files x every version bump and publish  + every stack branch). 

NOTES: 
* We may need additional fixes.
* This PR removes links, some of which we might want to format properly and re-add later.

### CODE REVIEW

File: docs/input-logstash.asciidoc [hunk 1]
```diff
@@ -91,7 +91,7 @@ NOTE: Client-certificate verification does _not_ verify identity claims on the p
 ===== Security: Credentials
 
 You can also configure this plugin to require a specific username/password be provided by configuring <<plugins-{type}s-{plugin}-username>> and <<plugins-{type}s-{plugin}-password>>.
-Doing so requires connecting <<plugins-outputs-logstash>> clients to provide matching <<plugins-outputs-{plugin}-username>> and <<plugins-outputs-{plugin}-password>>.
+Doing so requires connecting <<plugins-outputs-logstash>> clients to provide matching <<plugins-{type}s-{plugin}-username>> and <<plugins-{type}s-{plugin}-password>>.
```
Comment (karenzone) [inline docs/input-logstash.asciidoc]:
```suggestion
Doing so requires connecting {logstash-ref}/plugins-outputs-logstash.html[Logstash Output Plugin] clients to provide matching <<plugins-{type}s-{plugin}-username>> and <<plugins-{type}s-{plugin}-password>>.
```

File: docs/output-logstash.asciidoc [hunk 2]
```diff
@@ -90,7 +90,7 @@ If the downstream input plugin is configured to request or require client authen
 [id="plugins-{type}s-{plugin}-config-credentials"]
 ===== Security: Credentials
 
-If the downstream <<plugins-inputs-logstash>> is configured to require <<plugins-inputs-logstash-username>> and <<plugins-inputs-logstash-password>>,
+If the downstream <<plugins-inputs-logstash>> is configured to require <<plugins-{type}s-{plugin}-username>> and <<plugins-{type}s-{plugin}-password>>,
```
Comment (karenzone) [inline docs/output-logstash.asciidoc]:
```suggestion
If the downstream {logstash-ref}/plugins-inputs-logstash.html[Logstash input plugin] is configured to require <<plugins-{type}s-{plugin}-username>> and <<plugins-{type}s-{plugin}-password>>,
```

File: docs/input-logstash.asciidoc [hunk 3]
```diff
@@ -91,7 +91,7 @@ NOTE: Client-certificate verification does _not_ verify identity claims on the p
 ===== Security: Credentials
 
 You can also configure this plugin to require a specific username/password be provided by configuring <<plugins-{type}s-{plugin}-username>> and <<plugins-{type}s-{plugin}-password>>.
-Doing so requires connecting <<plugins-outputs-logstash>> clients to provide matching <<plugins-outputs-{plugin}-username>> and <<plugins-outputs-{plugin}-password>>.
+Doing so requires connecting `logstash-output` plugin clients to provide matching <<plugins-{type}s-{plugin}-username>> and <<plugins-{type}s-{plugin}-password>>.
```
Comment (karenzone) [inline docs/input-logstash.asciidoc]:
```suggestion
Doing so requires connecting `logstash-output` plugin clients to provide matching `username` and `password`. 
```

File: docs/input-logstash.asciidoc [hunk 4]
```diff
@@ -240,7 +240,7 @@ A password or passphrase of the <<plugins-{type}s-{plugin}-ssl_key>>.
 * There is no default value for this setting.
 
 Username for password-based authentication.
-When this input plugin is configured with a `username`, it also requires a <<plugins-{type}s-{plugin}-password>>, and any upstream <<plugins-outputs-logstash>> must also be configured with a matching `username`/`password` pair.
+When this input plugin is configured with a `username`, it also requires a <<plugins-{type}s-{plugin}-password>>, and any upstream `logstash-output` plugin must also be configured with a matching `username`/`password` pair.
```
Comment (karenzone) [inline docs/input-logstash.asciidoc]:
```suggestion
When this input plugin is configured with a `username`, it also requires a `password`, and any upstream `logstash-output` plugin must also be configured with a matching `username`/`password` pair.
```

File: docs/output-logstash.asciidoc [hunk 5]
```diff
@@ -63,7 +63,7 @@ output {
 [id="plugins-{type}s-{plugin}-config-connecting"]
 ===== Configuration Concepts
 
-This output plugin needs to be configured to connect to a <<plugins-inputs-logstash>> by specifying its <<plugins-{type}s-{plugin}-hosts>>.
+Configure this output plugin needs to connect to a {logstash-ref}/plugins-inputs-logstash.html[Logstash input plugin] by specifying its <<plugins-{type}s-{plugin}-hosts>>.
```
Comment (karenzone) [inline docs/output-logstash.asciidoc]:
```suggestion
Configure this output plugin to connect to a {logstash-ref}/plugins-inputs-logstash.html[Logstash input plugin] by specifying its `hosts`.
```

File: docs/output-logstash.asciidoc [hunk 6]
```diff
@@ -90,7 +90,7 @@ If the downstream input plugin is configured to request or require client authen
 [id="plugins-{type}s-{plugin}-config-credentials"]
 ===== Security: Credentials
 
-If the downstream <<plugins-inputs-logstash>> is configured to require <<plugins-inputs-logstash-username>> and <<plugins-inputs-logstash-password>>,
+If the downstream `logstash-input` plugin is configured to require <<plugins-{type}s-{plugin}-username>> and <<plugins-{type}s-{plugin}-password>>,
```
Comment (karenzone) [inline docs/output-logstash.asciidoc]:
```suggestion
If the downstream `logstash-input` plugin is configured to require `username` and `password`,
```

**Review by mashhurs — APPROVED** [review]
LGTM!
Thanks for fixing this.

**Comment by karenzone** [comment]
Published at https://rubygems.org/gems/logstash-integration-logstash/versions/0.0.5-java

**Pull request merged**